airgun/entities/task.py

```diff
@@ -20,6 +20,22 @@ def read(self, entity_name, widget_names=None):
         view = self.navigate_to(self, 'Details', entity_name=entity_name)
         return view.read(widget_names=widget_names)
 
+    def set_chart_filter(self, char_name, index=None):
```

Comment:
Can you rename this parameter to `chart_name`?

`char_name` suggests you mean name of character, which is not the case here.